modules/broadcast/src/backend/daemon.ts

```diff
@@ -17,35 +17,11 @@ export default async (botId: string, bp: SDK, db: Database) => {
   }, 1000)
 
   const _sendBroadcast = Promise.method((botId, row) => {
-    let dropPromise = Promise.resolve(false)
-
-    if (row.filters) {
-      dropPromise = Promise.mapSeries(JSON.parse(row.filters), (filter: string) => {
-        let fnBody = filter.trim()
-        if (!/^return /i.test(fnBody)) {
-          fnBody = 'return ' + fnBody
-        }
-
-        const fn = new Function('bp', 'userId', 'platform', fnBody)
-        return Promise.method(fn)(bp, row.userId, row.platform)
-      }).then(values => {
-        return _.some(values, v => {
-          if (!_.isBoolean(v)) {
-            bp.logger.warn('Filter returned something other ' + 'than a boolean (or a Promise of a boolean)')
-          }
-
-          return typeof v !== 'undefined' && v !== null && v !== true
-        })
-      })
-    }
```

Comment:
I still think filters are an important part of broadcast. Without them, impossible to ignore users who haven't spoke to the bot in the last month, and also impossible to filter unwanted channels (eg: api when benchmarking creates a ton of users).